grammars/lsl.cson

```diff
@@ -61,6 +61,18 @@
                 'name': 'punctuation.definition.comment.lsl'
         'match': '(//).*$\\n?'
         'name': 'comment.line.double-slash.lsl'
+    'pragma_inline':
```

Comment:
I just tweaked similar examples I found online and worked for me. Maybe they don't follow some sort of best practices, but I've not had any issues with these changes.